### PR TITLE
add last_git_commit action

### PR DESCRIPTION
**Pull request by ngutman**
Return information about the last git commit, we use it to create a custom message when releasing a Crashlytics build.

Might be useful to other people

### CODE REVIEW

File: lib/fastlane/actions/last_git_commit.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      def self.return_value
+        "Returns the following dict: {author: \"Author\", message: \"commit message\"}"
+      end
+
+      def self.author
+        "ngutman"
+      end
+
+      def self.is_supported?(platform)
+        platform == :ios
```
Comment (KrauseFx) [inline lib/fastlane/actions/last_git_commit.rb]: Can you replace this with just `true`, as this works on any platform

**Comment by KrauseFx** [comment]
Thank you so much @ngutman, that's a great addition. I added one comment. Also, could you add a short description and a very simple example usage to the `Actions.md` document?

**Comment by ngutman** [comment]
@KrauseFx added a description to `Actions.md` and supporting all platforms now, thanks!

**Comment by KrauseFx** [comment]
Looks great, thank you for your contribution @ngutman :+1:

**Pull request merged**